README.md

```diff
@@ -1,4 +1,5 @@
-[![Build Status](https://travis-ci.org/jscs-dev/node-jscs.svg?branch=master)](https://travis-ci.org/jscs-dev/node-jscs)
+﻿[![Build Status](https://travis-ci.org/jscs-dev/node-jscs.svg?branch=master)](https://travis-ci.org/jscs-dev/node-jscs)
```

Comment:
@mikesherov, yeah that was both CRLF and BOM. Fixed by 3477d8a76b2d3dfeee16d0e2c6610f169791f12c.